router.go

```diff
@@ -128,6 +128,36 @@ func createVersion(c echo.Context) (err error) {
 	return c.JSON(http.StatusCreated, ver)
 }
 
+func getPendingVersions(c echo.Context) (err error) {
+	if err = checkAuthorized(c); err != nil {
+		return err
+	}
+
+	opts := &registry.AppOptions{}
+	if err = c.Bind(opts); err != nil {
```

Comment:
Since it is a GET route, a query-parameter may more appropriate here than a request with a JSON body.